lms/djangoapps/instructor/management/commands/dump_grades.py

```diff
@@ -1,76 +0,0 @@
-#!/usr/bin/python
-"""
```

Comment:
Cool. I pinged in the devops chat room but at-team doesn't seem to be working. So perhaps @feanil can answer, or find out.